Lesson33/gozde/lesson33-app/src/Exercise4/index.jsx

```diff
@@ -0,0 +1,20 @@
+
+import React from "react";
```

Comment:
In current version of React, you don't need to add this import any more. Feel free to remove this line completely.